package edu.lessons.dayB.exceptions.ex1;

class A {
	public static void m4() //throws Exception 
	{
		Exception e1=new Exception("Error 1");
		//Inner Exception
		//e1 is nested inside e2
		Exception e2=new Exception("Error 2",e1);
		//throw e2;
	}
	public static  void testM4()
	{
		m4();
	}

}
